### ISSUE TITLE
[Issue]: 10.8.0 - DVR cannot record with HDHR-US legacy tuner

### ISSUE THREAD

**Issue by SenorSmartyPants** [jellyfin/jellyfin#7572]
### Please describe your bug

1. Go to live TV guide
2. Select any program
3. Click Record
4. Nothing happens

Logs report MediaBrowser.Controller.LiveTv.LiveTvConflictException (see logs below). No other devices are attempting to use tuners.

### Jellyfin Version

10.8.0

### if other:

beta1

### Environment

```markdown
- OS: Armbian
- Virtualization: Docker
- Clients: Browser, Android
- Browser: Chrome 100
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
[2022-04-08 15:26:25.557 -05:00] [INF] [82] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Recording timer fired for "Around the World in 80 Days".
[2022-04-08 15:26:25.702 -05:00] [INF] [82] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "hdhr_21.1"
[2022-04-08 15:26:25.712 -05:00] [INF] [82] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel "hdhr_21.1"
[2022-04-08 15:26:25.759 -05:00] [INF] [82] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: GetChannelStream: channel id: "hdhr_21.1". stream id: "native_1dabc362276bf3630c2857a6e5bcc49a_e73af4f2996429a3c39c05d809b22ec9" profile: "native"
[2022-04-08 15:26:25.832 -05:00] [INF] [82] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Opening HDHR UDP Live stream from "192.168.1.49"
[2022-04-08 15:26:25.961 -05:00] [INF] [82] Emby.Server.Implementations.LiveTv.LiveTvManager: New recording scheduled
[2022-04-08 15:26:25.961 -05:00] [WRN] [82] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://rockpi:8096/LiveTv/Timers" to "192.168.1.24" in 0:00:00.6615583 with Status Code 204
[2022-04-08 15:26:26.072 -05:00] [ERR] [87] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening live stream:
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
[2022-04-08 15:26:26.123 -05:00] [ERR] [87] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening tuner
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[2022-04-08 15:26:26.147 -05:00] [ERR] [87] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Error recording to "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.RecordStream(TimerInfo timer, DateTime recordingEndDate, ActiveRecordingInfo activeRecordingInfo)
[2022-04-08 15:26:26.155 -05:00] [INF] [87] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Triggering refresh on "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
[2022-04-08 15:26:26.158 -05:00] [INF] [87] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Refreshing recording parent "/storage/media/Videos/DVR/TV"
[2022-04-08 15:26:26.159 -05:00] [INF] [87] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Retrying recording in 60 seconds.
[2022-04-08 15:26:26.169 -05:00] [INF] [87] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Creating recording timer for "2ab56a0b178b47f8bc1d88f1fda31d91", "Around the World in 80 Days". Timer will fire in "0.9998392416666667" minutes
[2022-04-08 15:26:36.766 -05:00] [WRN] [108] Jellyfin.Networking.Manager.NetworkManager: IPv6 is disabled in Jellyfin, but enabled in the OS. This may affect how the interface is selected.
[2022-04-08 15:26:38.399 -05:00] [WRN] [87] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://rockpi:8096/Items/Counts" to "172.18.0.5" in 0:00:00.6381314 with Status Code 200
[2022-04-08 15:27:26.164 -05:00] [INF] [108] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Recording timer fired for "Around the World in 80 Days".
[2022-04-08 15:27:26.169 -05:00] [INF] [108] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "hdhr_21.1"
[2022-04-08 15:27:26.170 -05:00] [INF] [108] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel "hdhr_21.1"
[2022-04-08 15:27:26.171 -05:00] [INF] [108] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: GetChannelStream: channel id: "hdhr_21.1". stream id: "native_1dabc362276bf3630c2857a6e5bcc49a_e73af4f2996429a3c39c05d809b22ec9" profile: "native"
[2022-04-08 15:27:26.174 -05:00] [INF] [108] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Opening HDHR UDP Live stream from "192.168.1.49"
[2022-04-08 15:27:26.181 -05:00] [ERR] [73] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening live stream:
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
[2022-04-08 15:27:26.186 -05:00] [ERR] [73] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening tuner
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[2022-04-08 15:27:26.213 -05:00] [ERR] [73] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Error recording to "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.RecordStream(TimerInfo timer, DateTime recordingEndDate, ActiveRecordingInfo activeRecordingInfo)
[2022-04-08 15:27:26.218 -05:00] [INF] [73] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Triggering refresh on "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
[2022-04-08 15:27:26.222 -05:00] [INF] [73] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Refreshing recording parent "/storage/media/Videos/DVR/TV"
[2022-04-08 15:27:26.223 -05:00] [INF] [73] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Retrying recording in 60 seconds.
[2022-04-08 15:27:26.235 -05:00] [INF] [73] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Creating recording timer for "2ab56a0b178b47f8bc1d88f1fda31d91", "Around the World in 80 Days". Timer will fire in "0.9998119733333334" minutes
[2022-04-08 15:28:26.222 -05:00] [INF] [95] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Recording timer fired for "Around the World in 80 Days".
[2022-04-08 15:28:26.227 -05:00] [INF] [95] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "hdhr_21.1"
[2022-04-08 15:28:26.227 -05:00] [INF] [95] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel "hdhr_21.1"
[2022-04-08 15:28:26.228 -05:00] [INF] [95] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: GetChannelStream: channel id: "hdhr_21.1". stream id: "native_1dabc362276bf3630c2857a6e5bcc49a_e73af4f2996429a3c39c05d809b22ec9" profile: "native"
[2022-04-08 15:28:26.231 -05:00] [INF] [95] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Opening HDHR UDP Live stream from "192.168.1.49"
[2022-04-08 15:28:26.235 -05:00] [ERR] [132] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening live stream:
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
[2022-04-08 15:28:26.240 -05:00] [ERR] [132] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening tuner
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[2022-04-08 15:28:26.243 -05:00] [ERR] [132] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Error recording to "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.RecordStream(TimerInfo timer, DateTime recordingEndDate, ActiveRecordingInfo activeRecordingInfo)
[2022-04-08 15:28:26.248 -05:00] [INF] [132] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Triggering refresh on "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
[2022-04-08 15:28:26.253 -05:00] [INF] [132] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Refreshing recording parent "/storage/media/Videos/DVR/TV"
[2022-04-08 15:28:26.253 -05:00] [INF] [132] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Retrying recording in 60 seconds.
[2022-04-08 15:28:26.266 -05:00] [INF] [132] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Creating recording timer for "2ab56a0b178b47f8bc1d88f1fda31d91", "Around the World in 80 Days". Timer will fire in "0.9998001366666667" minutes
[2022-04-08 15:28:36.659 -05:00] [WRN] [41] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://rockpi:8096/LiveTv/Recordings?userId=5c0c92019ab2454a8bdcc7aba9abcdd6&Limit=12&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&EnableTotalRecordCount=false&IsInProgress=true" to "192.168.1.24" in 0:00:00.7044824 with Status Code 200
[2022-04-08 15:28:37.487 -05:00] [WRN] [101] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://rockpi:8096/Users/5c0c92019ab2454a8bdcc7aba9abcdd6/Items/Latest?Limit=16&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo%2CPath&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CThumb&ParentId=71a73d06363d25418d8bb64613ea98f6" to "192.168.1.24" in 0:00:01.5179806 with Status Code 200
[2022-04-08 15:28:39.115 -05:00] [WRN] [117] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://rockpi:8096/Shows/NextUp?Limit=24&Fields=PrimaryImageAspectRatio%2CDateCreated%2CBasicSyncInfo%2CPath%2CMediaSourceCount&UserId=5c0c92019ab2454a8bdcc7aba9abcdd6&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&EnableTotalRecordCount=false&DisableFirstEpisode=false&NextUpDateCutoff=2021-04-08T20%3A27%3A38.569Z&EnableRewatching=true" to "192.168.1.24" in 0:00:03.1622985 with Status Code 200
[2022-04-08 15:29:26.253 -05:00] [INF] [86] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Recording timer fired for "Around the World in 80 Days".
[2022-04-08 15:29:26.257 -05:00] [INF] [86] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "hdhr_21.1"
[2022-04-08 15:29:26.258 -05:00] [INF] [86] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel "hdhr_21.1"
[2022-04-08 15:29:26.259 -05:00] [INF] [86] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: GetChannelStream: channel id: "hdhr_21.1". stream id: "native_1dabc362276bf3630c2857a6e5bcc49a_e73af4f2996429a3c39c05d809b22ec9" profile: "native"
[2022-04-08 15:29:26.262 -05:00] [INF] [86] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Opening HDHR UDP Live stream from "192.168.1.49"
[2022-04-08 15:29:26.267 -05:00] [ERR] [134] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening live stream:
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
[2022-04-08 15:29:26.272 -05:00] [ERR] [134] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening tuner
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[2022-04-08 15:29:26.275 -05:00] [ERR] [134] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Error recording to "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.RecordStream(TimerInfo timer, DateTime recordingEndDate, ActiveRecordingInfo activeRecordingInfo)
[2022-04-08 15:29:26.279 -05:00] [INF] [134] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Triggering refresh on "/storage/media/Videos/DVR/TV/Around the World in 80 Days/Season 1/Around the World in 80 Days S01E01.ts"
[2022-04-08 15:29:26.284 -05:00] [INF] [134] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Refreshing recording parent "/storage/media/Videos/DVR/TV"
[2022-04-08 15:29:26.284 -05:00] [INF] [134] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Retrying recording in 60 seconds.
[2022-04-08 15:29:26.297 -05:00] [INF] [134] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Creating recording timer for "2ab56a0b178b47f8bc1d88f1fda31d91", "Around the World in 80 Days". Timer will fire in "0.9997956116666666" minutes
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SenorSmartyPants** [comment]
Info on my tuner
![image](https://user-images.githubusercontent.com/991618/162527972-c35e40d0-2ba4-446f-aedd-ecf4731c2c6a.png)

**Comment by SenorSmartyPants** [comment]
Pulling 10.7.7 tag - Legacy HDHR tuner works
10.8.0-beta1 - Legacy HDHR tuner does not work.

Trying to figure out what commit might have caused the problem.... Not having great luck with that.

**Comment by SenorSmartyPants** [comment]
@Bond-009 seems like you might have done the most in this area. 

I've traced into CheckTunerAvailability(NetworkStream stream, int tuner, CancellationToken cancellationToken). There seems to be data coming back to the buffer, but whatever the response is it does not say the tuner is available. Even though both are available. Is this checked differently for Legacy HDHR?

I'm happy to run any test using my HDHR.

**Comment by SenorSmartyPants** [comment]
This should be labeled as a regression, since pre-10.8 I could record fine with this hardware.

**Comment by mattsteg** [comment]
I have the same issue - currently running on FreeBSD (so not officially supported) but getting the same error message.

Had you found any resolution or is there any additional information I can help with?

**Comment by ruthara** [comment]
Duplicate of #7154

**Comment by SenorSmartyPants** [comment]
Closed with #8726
